src/summary/profile/contacts-card.component.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import SummaryCard from "../cards/summary-card.component";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import VerticalLabelValue from "../cards/vertical-label-value.component";
+import { match } from "react-router";
+
+export default function ContactsCard(props: ContactsCardProps) {
+  function getAddress(address: fhir.Address) {
+    return `${address.city}, ${address.country} \n `;
+  }
+
+  return (
+    <SummaryCard name="Contact & Address" match={props.match}>
+      {props.patient && props.patient.address ? (
+        props.patient.address.map(address => (
+          <SummaryCardRow key={address.id}>
+            <SummaryCardRowContent>
+              <VerticalLabelValue
+                label={address.use}
+                value={getAddress(address)}
+              />
+            </SummaryCardRowContent>
+          </SummaryCardRow>
+        ))
+      ) : (
+        <SummaryCardRow>
+          <SummaryCardRowContent>None</SummaryCardRowContent>
```

Comment:
Is this in the designs? We've been using emdash for when content is missing -- any reason to deviate from that here?

Or even should we just not render a SummaryCardRow at all when there is no address?

Reply:
This has not been addressed in the designs. I  think that using em-dash might be better to explicitly let a user know that there is no address/contact